tokens/src/components/Toast.json

```diff
@@ -41,11 +23,41 @@
           }]
         }
       }
-    },
-    "container": {
-      "gutter": {
-        "lg": { "value": "1.25rem", "source": "$toast-container-gutter-lg" },
-        "sm": { "value": ".625rem", "source": "$toast-container-gutter-sm" }
+    }
+  },
+  "typograthy": {
+    "toast": {
+      "font-size": { "value": ".875rem", "source": "$toast-font-size" }
+    }
+  },
+  "elevation": {
+    "toast": {
+      "box-shadow": {
+        "value": "0 1.25rem 2.5rem rgba(0, 0, 0, .15), 0 .5rem 3rem rgba(0, 0, 0, .15)",
+        "source": "$toast-box-shadow"
+      }
+    }
+  },
+  "size": {
+    "toast": {
+      "max-width": { "value": "400px", "source": "$toast-max-width" },
+      "border": {
+        "width": { "value": "1px", "source": "$toast-border-width" },
+        "radius": { "value": ".25rem", "source": "$toast-border-radius" }
+      },
+      "container": {
```

Comment:
I think this belongs to `spacing` category? These variable are used to positional properties (`top`, `bottom` etc.)

Reply:
moved, thanks